Tool/examples/0.4-DRAFT/validata_tools.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I do not see this examples linked from the Tool profile description page as it happened with the examples for the Protein profile.

Reply:
I'm sorry, I don't understand your comment. Can you explain a bit more what you mean?

Reply:
It was about having a link to the examples but then those links would need to be maintained. No action required here